crates/topos/src/options.rs

```diff
@@ -29,11 +34,29 @@ pub(crate) struct Opt {
     )]
     pub(crate) verbose: u8,
 
+    /// Home directory for the configuration
+    #[arg(
+    long,
+    env = "TOPOS_HOME",
+    default_value = get_default_home(),
+    global = true
+    )]
+    pub(crate) home: PathBuf,
+
     #[command(subcommand)]
     pub(crate) commands: ToposCommand,
 }
 
-#[derive(Subcommand, Debug)]
+/// If no path is given for the --home argument, we use the default one
+/// ~/.config/topos for a UNIX subsystem
+fn get_default_home() -> OsString {
+    let mut home = dirs::home_dir().unwrap();
+    home.push(".config");
```

Comment:
if we aim to use config folder, we can use `config_dir`  instead of `home_dir`

Reply:
This gives us a different set of problems with sanitizing for the snapshot tests, because every OS has a different `config` path. I don't see the advantage of adding more sanitizing tests if we can just more tightly control where the default path is.